dsgrid/cloud/s3_storage_interface.py

```diff
@@ -27,9 +27,12 @@ def __init__(self, local_path, remote_path, uuid, user, profile):
         self._local_filesystem = LocalFilesystem()
         self._s3_filesystem = S3Filesystem(remote_path, profile)
 
-    def _sync(self, src, dst, exclude=None):
+    def _sync(self, src, dst, exclude=None, is_file=False):
         start = time.time()
-        sync_command = f"aws s3 sync {src} {dst} --profile {self._s3_filesystem._profile}"
+        if is_file:
+            sync_command = f"aws s3 cp {src} {dst} --profile {self._s3_filesystem._profile}"
```

Comment:
Here and below with `_bucket` you are referencing private attributes of `_s3_filesystem`. Can you make those public properties to avoid this?
```
@property
def profile(self):
    return self._profile
```